.github/workflows/nodejs-ci.yml

```
@@ -10,9 +13,9 @@ jobs:
 
     steps:
 
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
 
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v4
         with:
           node-version: 16.x
```

```suggestion
          node-version: 22.x
```
But it may require some development dependencies to be updated as well, very similar to: https://github.com/Borewit/peek-readable/pull/690/files